SDL/Quintuplet.cu

```diff
@@ -122,8 +122,30 @@ void SDL::quintuplets::freeMemory(cudaStream_t stream)
 #endif
 cudaStreamSynchronize(stream);
 }
-
 //TODO:Reuse the track candidate one instead of this!
+__global__ void SDL::createEligibleModulesListForQuintupletsGPU(struct modules& modulesInGPU,struct triplets& tripletsInGPU, unsigned int maxQuintuplets, cudaStream_t stream,struct objectRanges& rangesInGPU)
+{
+    __shared__ int nEligibleT5Modulesx;
+    nEligibleT5Modulesx =-1;
+
+
+
+    //start filling
+    int gid = blockIdx.x * blockDim.x + threadIdx.x;
+    int np = gridDim.x * blockDim.x;
+    for(uint16_t i = gid; i < *modulesInGPU.nLowerModules; i+= np)
+    {
+        //condition for a quintuple to exist for a module
+        //TCs don't exist for layers 5 and 6 barrel, and layers 2,3,4,5 endcap
```

Comment:
So this part of the code is looking for modules that have T3s. in the barrel you can't have a T3 start on layer 5 or 6 because you need to go over 3 layers and there are only 6 in the barrel. For the endcap, I think the layer numbers are 1,7,8 or something like that. So this is skipping all the modules we know we don't want or need.

Reply:
Ok, I was confused because the comment is referring to TCs, not T3s, so I was wondering how that was connected to what we are computing here.